### ISSUE TITLE
CC components not showing in Product Detail Page

### ISSUE THREAD

**Issue by olivercastro** [magesuite/magesuite#31]
Preconditions (*)

1. Magento 2.3.3 + Sample data
2. creativestyle/magesuite v3.0.0

Steps to Reproduce:

1. Go to Catalog > Inventory > Products
2. Edit product
3. Go to Product's Content Constructor 
![Screenshot from 2019-12-15 11-12-40](https://user-images.githubusercontent.com/4221460/70857587-847c9280-1f2c-11ea-9263-a6d8551282fa.png)
4. Click Save

Expected result (*)

1. CC component should show on Product Detail page

Actual Result (*)
1. CC component not showing
![Screenshot from 2019-12-15 11-19-01](https://user-images.githubusercontent.com/4221460/70857598-bdb50280-1f2c-11ea-83ec-810416a451b0.png)

**Comment by marcelomartinsIT** [comment]
As of writing this post, MageSuite isn't fully compatible with Magento 2.3.3. Do expect bugs until it is properly fixed.

**Comment by maxim-torba** [comment]
any update on this?

i'm using "creativestyle/magesuite": "^5.0.0", with  magento version "2.3.4",

**Comment by diwipl** [comment]
CC should work without any issues on product page. Are you sure you're checking correct store view configuration ? Maybe there are some components defined for "All store views" and no components defined for specific store view ?

**Comment by ikslavok** [comment]
@diwipl I have a similar issue. I added magesuite to try it, but can't use CC anywhere. In the readme, you say all frontend features can't be used on custom themes. Does it mean that if I add magesuite's CC to my already live site, that already has a theme, I won't be able to use it?

**Comment by diwipl** [comment]
@MrKovalski Yes, unless you write your own CSS and JS then most of the components will be useless without our theme.

**Comment by ikslavok** [comment]
@diwipl Is there a list of modules which can be used without Creativeshop? I've seen there are quite few modules that are still very useful without Creativeshop.

**Comment by koria** [comment]
any update on this? 

 "creativestyle/magesuite": "^6.0.0"
 "magento": "2.4.0"

components not showing in Product Detail Page

**Comment by salvolf** [comment]
Same here!

**Comment by p-makowski** [comment]
M2.4.7-p4 without simple data and I got the issue as well. 

CC works correctly on CMS pages and on category page, is just not outputted on product pages. 
It is very fresh installation, with some dummy, manually created products. 
No errors in any of consoles. 

@diwipl Do you have any hints on where is a good starting point for an investigation?

**Comment by p-makowski** [comment]
After a few evenings with debugging I figured out that the problem with missing CC components (as well as product attributes and description) was caused by _Default Product Layout_ being set to `product-full-width`. 
For some reason it does not include CC container. 

![Image](https://github.com/user-attachments/assets/ea360c85-cc12-4538-9292-ef9eff82d4ff)

Works when it is changed to "null" (in store configuration for new products only, on product level valid ofc for given product)

 
![Image](https://github.com/user-attachments/assets/62e60f95-0acb-49a0-ae1c-60b95f5dd43e)
